tests/transports-governance/conftest.py

```
@@ -0,0 +1,668 @@
+        try:
+            response = self.session.request(method, url, **kwargs)
+            return response
+        except requests.exceptions.RequestException as e:
+            pytest.fail(f"Request failed: {method} {url} - {str(e)}")
+
+    # Virtual Key operations
+    def list_virtual_keys(self, **params) -> requests.Response:
+        """List all virtual keys"""
+        return self.request("GET", "/virtual-keys", params=params)
+
+    def create_virtual_key(self, data: Dict[str, Any]) -> requests.Response:
```

Ruff (0.12.2)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)